File: src/StockMarket.Domain/ExitOrder.cs
using SharedKernel;
using StockMarket.Domain.Enums;
using System;

namespace StockMarket.Domain
{
    public class ExitOrder : EntityBase
    {
        public int EntryOrderId { get; set; }
        public DateTime Date { get; set; }
        public decimal Price { get; set; }
        public int OrderQuantity { get; set; }
        public int ExecutedQuantity { get; set; }
        public DateTime? LastModifiedDate { get; set; }
        public string BrockerRef { get; set; }
        public OrderStatus Status { get; set; }

        public EntryOrder EntryOrder { get; set; }
    }
}

File: src/StockMarketWebClient/src/app/shared/models/ExitOrder.ts
import { EntryOrder } from './EntryOrder';

export interface ExitOrder {
    id: number;
    isDirty: boolean;
    entryOrderId: number;
    date: Date;
    price: number;
    orderQuantity: number;
    executedQuantity: number;
    lastModifiedDate: Date;
    brockerRef: string;
    status: string;

    entryOrder: EntryOrder;
}